postgres_lambda_strategy/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "aws_region" {
+  description = "AWS Region"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "db_config" {
+  description = "Connection configuration for your Postgres Database"
+  type        = object({ host = string, port = number, user = string, pass = string })
+}
+
+variable "security_group_id" {
+  description = "Security group for the lambda"
+  type        = string
+}
+
+variable "sym_org" {
```

Comment:
For the sake of example simplicity, we are trying to reduce the number of variables used, as these examples are not intended to be drag and drop solutions to be copied, but as reference material that can be understood at a glance

Let's remove this variable, for consistency with the other examples